basedtyping/__init__.py

```diff
@@ -100,70 +105,81 @@ def __instancecheck__(self, instance: object) -> bool:
 
 class OrigClass(Protocol):
     __args__: tuple[type, ...]
-    """the reified type(s)"""
+    """The reified type(s)"""
     __parameters__: tuple[TypeVar, ...]
-    """any unbound ``TypeVar``s (this should always be empty by the time the ``ReifiedGeneric`` is instantiated)"""
+    """Any unbound ``TypeVar``s (this should always be empty by the time the 
+    ``ReifiedGeneric`` is instantiated)
+    """
 
 
 class ReifiedGenericError(TypeError):
     pass
 
 
-class NotReifiedError(ReifiedGenericError):
-    """raised when a ``ReifiedGeneric`` is instanciated without using a generic alias
+class MissingTypeParametersError(ReifiedGenericError):
+    """Raised when a ``ReifiedGeneric`` is instantiated without passing type parameters.
 
-    ie. ``foo: Foo[int] = Foo()`` instead of ``foo = Foo[int]()``"""
+    ie: ``foo: Foo[int] = Foo()`` instead of ``foo = Foo[int]()``
+    """
 
 
-class MissingTypeParametersError(ReifiedGenericError):
-    """Raised when a ``ReifiedGeneric`` is instanciated with a non-reified ``TypeVar`` instead of a type parameter
+class NotReifiedError(ReifiedGenericError):
```

Comment:
werent u gonna rename this because `NotReified` also applies to  the scenario covered by `MissingTypeParametersError`